README.md

```diff
@@ -51,25 +77,40 @@ The application will simply browse through a catalog of products. It will suppor
 The online shop has a "shopping cart" functionality:
 - The user may add items into the cart,
 - He may increment and decrement the quantity of each product or even remove a product completely from the cart,
-- Lastly, he may checkout the cart and place an order (resulting in the creation of an order in the backend).
+- Lastly, he can check out the cart and place an order (resulting in the creation of an order in the backend).
 
-Mockups describing the user interface structure can be found in the [mockups](./mockups/mockups.pdf) folder.
+Mockups describing the user interface structure can be found in the [mockup's](./mockups/mockups.pdf) folder.
 These mockups should be used as a guideline, but improvements or deviations from them is allowed.
 
+### Notes:
+- If you find any link broken, **please** inform your mentor to give you an alternative
+- Try to speed up the videos to *1.5x/2x* if you find them too slow
+- Try to connect to the Backend from the Spring Training (you might need to change the port of your requests)
+
 ## Timeline
 This timeline is just for guidance, take time to understand the concepts before moving on.
 
 - **Day 1**: Chapter 0, Chapter 1, Chapter 2
-- **Day 2**: Chapter 3, Chapter 4
-- **Day 3**: Chapter 5, Chapter 6
-- **Day 4**: Chapter 7, Chapter 8
-- **Day 5**: Review Remarks && Corrections, Optional Chapters
+- **Day 2**: Chapter 3, Chapter 4, **Open a Pull Request**
+- **Day 3**: Fix Review Remarks, Chapter 5, Chapter 6
+- **Day 4**: Chapter 7, Chapter 8, **Open a Pull Request**
+- **Day 5**: Fix Review Remarks && Corrections, Optional Chapters
 
 ## 0. HTML and CSS/SCSS Basics [1h - 2h]
 
 Goal: refresh your knowledge about HTML and CSS basic concepts.
 
-Required Reading:
+#### Required:
+
+*Videos*
+
+1. [HTML in 100 Seconds](https://www.youtube.com/watch?v=ok-plXXHlWw)
+2. [CSS in 100 Seconds](https://www.youtube.com/watch?v=OEV8gMkCHXQ)
+3. [JavaScript in 100 Seconds](https://www.youtube.com/watch?v=DHjqpvDnNGE)
+4. [TypeScript in 100 Seconds](https://www.youtube.com/watch?v=zQnBQ4tB3ZA)
```

Comment:
Maybe add a why react sucks video as well to spite the react trainers. /s

Reply:
:))) I hate things about both frameworks